src/js/common/utils/helpers.js

```diff
@@ -66,6 +66,17 @@ export function getInactivePages(pages, data) {
   return pages.filter(page => !isActivePage(page, data));
 }
 
+export function getCurrentFormStep(chapters, location) {
```

Comment:
This code was being used in HCA to handle some NavHeader logic. I knew I would need it for VA Letters, so I extracted it into the common helpers, and then revised HCA to use this helper as well.